doc/globus/download.md

```diff
@@ -0,0 +1,75 @@
+
+![Globus Download Menu](GlobusDownloadMenu.png)
+
+## 3. Globus pre-requisites:
+
+### a. Globus Account is Required
+
+You may already be able to log in as a Globus user using your school or institution's federated login. Visit [www.globus.org](https://www.globus.org) and click "Login" at the top of the page, then check the list of participating authentication providers. If an institutional login is not an option, you can create a Globus Id at [globusid.org](https://www.globusid.org). For more information please see [How To Log In and Transfer Files with Globus ](https://docs.globus.org/guides/tutorials/manage-files/transfer-files/).
+
+### b. Globus Connect Personal 
+
+[Globus Connect Personal](https://www.globus.org/globus-connect-personal) is a free client offered by Globus. Linux, Mac and Windows versions are available.
+
+Once Globus Connect Personal is installed, you will be asked to log in using your Globus account. Globus protocol is based around the idea of transferring data between _collections_. Connect Personal will create a Globus Collection on your computer that can be used as the destination end point for data transfers. Note that it is also possible to transfer data to collections other than the one on your laptop - you may, for example, facilitate a transfer to an end point hosted by your school where you have write access. But for the purposes of this instruction, we are working on the assumption that you are downloading the data to your own computer. 
+
+Verify that your new collection is properly set up by clicking the "Web: Collection Details" in the menu (MacOS version shown):
+
+![Globus Connect Personal Collection View](GlobusPersonalConnect.png)
+
+This should take you to the Globus Web App page, confirming that your personal collection is now part of the Globus federation:
+
+![Globus Web App Collection View](GlobusWebAppCollection.png)
+
+Since you will be transferring data into your collection, make sure that "Write Access" is checked in the _Access_ panel under _Preferences_. Keep the "Shareable" box unchecked, unless you actually have a need to make your personal collection accessible to other Globus users.
+
+Your Personal Collection will be limited initially to your home directory, but any other directory or volume local to your system can be added in the same _Access_ panel. For example, if you don't have enough disk space on your laptop to download all the data needed, an external hard drive can be added to the collection. 
+
+## 4. Borealis Globus App
+
+The download process relies on an extra software component, [dataverse-globus](https://github.com/scholarsportal/dataverse-globus), a web tool that serves as an interface between Dataverse and Globus. It is being developed by Victoria Lubitch at Borealis (formerly Scholars Portal), the Canadian Dataverse Repository project. Clicking on the Globus Transfer icon in the _File Access_ menu in the Dataverse UI will redirect you to the Borealis app in a new tab.
```

Comment:
I don't want to remove Victoria's name at this point, because I've already run it by her; and I specifically asked, "is it ok, the way I credit the app/your effort?" It would feel weird to drop her name after that.

Reply:
(I'm not sure about the past tense either; I really want to emphasize the fact that the app is still pretty much under development/in beta)

Reply:
Perhaps was originally developed by? Also wanted to note that the app is now at https://github.com/gdcc/dataverse-globus rather than the scholarsportal repo.